C++ Programs/Stacks/postfix_evaluation.cpp

```diff
@@ -0,0 +1,64 @@
+#include <bits/stdc++.h>
+
+#define OPERAND 1
+#define OPERATOR 0
+
+// to check if the current character is an operand
+int isOperand(char oper);
+
+using namespace std;
+
+// DRIVER FUNCTION
+int main()
+{
+    // Input is the postfix expression
+    char postfix[] = "73426^*+*8+";
```

Comment:
Why don't you take the expression as an input, rather than always changing the code?

Also, Using `char` is good but from 'C++' point of view, it will be better if we use the features of the language to a larger extent.
Like we have a `string` class inside CPP, and that can be used instead of `char`.

Using `char` is not wrong, but it is not a good practise as we are using 'modern C++'. Let's utilize the language properly.